### ISSUE TITLE
Blueprint adjustments (2)

### ISSUE THREAD

**Issue by Ben8t** [kestra-io/kestra#1568]
### Feature description

- [ ] When clicking on the "back" arrow, users are moved to the blueprints homepage -> would be nice that when the user search, the "go back" button goes back to the search results.

- [ ] Too many space between source code and corresponding title

![image](https://github.com/kestra-io/kestra/assets/46634684/ab5071bb-3422-4f59-afc0-8fd4e65e184b)

- [ ] add the possibility to move the topology horizontaly/verticically. If possible always set horizontal in Blueprints main section.
- [ ] Keep the breadcrumb (like in other page headers)

**Comment by anna-geller** [comment]
it would be great to also add this:

- [ ] open a blueprint in a new tab - currently, you can only open it on the same page, making it difficult to inspect/browse  blueprints

**Comment by brian-mulier-p** [comment]
@Ben8t regarding the "too many space" between source code and editor, that's because there's a copy button in the editor header which creates this offsets. Should I overlap the copy button in top-right corner of the actual editor (eventually hiding part of the id / namespace if it is a really long one) ?

For the topology orientation, I replicated the behaviour of the topology view: if on blueprints standalone page you are able to switch your orientation but when side-by-side with the editor you can't change the orientation and it is taken from your local storage (ie your preferences)

**Comment by brian-mulier-p** [comment]
> it would be great to also add this:
> 
> * [ ]  open a blueprint in a new tab - currently, you can only open it on the same page, making it difficult to inspect/browse  blueprints

Do you mean CTRL click to open in a new browser tab ?

**Comment by Ben8t** [comment]
@brian-mulier-p 

>  regarding the "too many space" between source code and editor, that's because there's a copy button in the editor header which creates this offsets. Should I overlap the copy button in top-right corner of the actual editor (eventually hiding part of the id / namespace if it is a really long one) ?

Yes go for it, so long id/namespace would be something very rare coming from internal blueprint (so it's 0.0001% chance of occurring atm)
